### DISCUSSION TITLE
Action for getting test/file relative path

### DISCUSSION THREAD

**Discussion by vol0n** [intellij-rust/intellij-rust#10094] [Q&A] [answered]
I have a lot of run configurations with different parameters for running tests, and often I need to change the test name to run the test I want with needed parameters, but typing in test path every time is a little tedious. I was wondering maybe there is an action that gives me the name of file/test to run in rust format like (`a::b::c::tests::test_name`)? Or maybe intellij-rust exposes api, that can be used to implement this? (I am not sure if it's possible to create plugins based on top of intellij-rust)

**Comment by vlad20012** [comment, marked as answer]
It seems we don't have such an action, but we definitely know such a path for every test function (otherwise we couldn't create run configurations for them), so it it should be easy to implement, I guess.

**Comment by vlad20012** [comment]
Ideally, we should change a path to a test function (like `a::b::c::tests::test_name`) in every run configuration when the test function is renamed using `Shift` + `F6`. It's possible to implement (but much harder then making an action to get a path to a function) and seems to be the right way in this case
